relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -75,7 +74,7 @@ pub fn normalize_parsed_queries(
     // Insert placeholders that the SQL serializer cannot provide.
     let replaced = concatenated.replace("___UPDATE_LHS___ = NULL", "..");
 
-    Ok((replaced, original_ast))
+    Ok((replaced, parsed))
```

Comment:
This is fine because the returned SQL is only ever used to extract table names from it?

I think it would be good to document that the returned value from `scrub_span_description` contains a parsed and modified SQL. This is quite a call chain to go through to figure out.

Reply:
👍 I'll do a follow-up PR to simplify the call chain: parse & scrub the SQL in `tag_extraction`, then pass it into both `scrub_span_description` and `sql_action_from_query`.